### ISSUE TITLE
install.sh does not define WSAPIKEY

### ISSUE THREAD

**Issue by rstemmer** [rstemmer/musicdb#47]
After a fresh install, the WSAPIKEY in config.js and musicdb.ini is not set.
The script should generate a key.

```
Uncaught ReferenceError: WSAPIKEY is not defined
    <anonymous> http://127.0.0.1/musicdb/webui/config.js:8
```

**Comment by rstemmer** [comment]
"Fixed" (not yet tested, but will be included in the 7.0.0 release)

**Comment by rstemmer** [comment]
Done
